src/content/blog/2024/fastquorum_intro.mdx

```diff
@@ -0,0 +1,215 @@
+These molecular barcodes are commonly referred to as Unique Molecular Indexes, or UMIs.
+They are attached to a DNA source molecule to uniquely identify it. After amplification, the multiple observations can be compared vote on a consensus.
+Or form a quorum, if you will.
+
+        <a href="https://doi.org/10.1038/nprot.2014.170"><Image src={duplex_sequencing_schema} alt="Duplex Sequencing Schema from Kennedy et al 2014" height="300" /></a>
+
+Molecular barcodes can be added at various points in the library preparation process, including prior to or during amplification, for a single strand or to both strands of a double-stranded duplex molecule
+In particular, [Duplex Sequencing][duplex-sequencing-link] can be used to identify reads from both strands of a double-stranded source molecule, squashing strand-specific errors that can occur during library preparation (e.g. PCR errors, oxidative damage due to hybrid capture).
+
+This means UMIs can be found in the index/sample-barcoding reads (i7/i5 reads in Illumina parlance), or inline in the genomic/template reads themselves (R1/R2). Furthermore, there sometimes exists extra “spacer” sequence which should be removed prior to alignment and downstream analysis.
+
+        <Image src={explaining_umis} alt="Always Sunny in Philadelphia conspiracy meme with caption 'Explaining where UMI bases are found'" height="300" />
+
+We utilize a [Read Structure][read-structure-link] to describe how the bases in a sequencing run should be allocated into logical reads.
+It serves a similar purpose to the `--use-bases-mask` in Illumina's `bcltofastq` software, but provides some additional capabilities.
+A few examples from the [fgbio wiki][read-structure-examples-link] are shown below:
+
+1. A simple 2x150bp paired end run with no sample or molecular indices:
+
+-   `150T150T`
+-   `[+T, +T]`
+
+2. A 2x75bp paired end run with an 8bp I1 index read:
+
+-   `75T8B75T`
+-   `[+T, 8B, +T]`
+
+3. A 2x150bp paired end run with an 8bp I1 index read and an inline 6bp UMI in read 1:
+
+-   `8M142T8B150T`
+-   `[8M+T, 8B, +T]`
+
+4. A 2x150bp duplex sequencing run with dual sample-barcoding (I1 and I2) and both a 10bp UMI and 5bp monotemplate at the start of both R1 and R2:
+
+-   `10M5S135T8B8B10M5S135T`
+-   `[10M5S+T, 8B, 8B, 10M5S+T]`
+
+By utilizing the [fgbio toolkit][fgbio-homepage-link] and the [Read Structure][read-structure-link] description, [nf-core/fastquorum][nf-core-fastquorum-link] is able to support the diversity of molecular barcoding schemes.
+A few of commercially available are described below:
+
+| Verified | Assay                                                     | Company                     | Strand | Randomness | URL                                                                                                                                                                                 |
```

Comment:
Is it correct that all the kits are not verified? Otherwise seems like an unnecessary column and could just a caveat in italics below the table